jq.py

```diff
@@ -112,6 +112,9 @@ def jq_command(debugger, command, result, dict):
     
     if val.GetTypeName() == "Foundation.Data":
         val_string = outputDataAsString(debugger, args[1])
+        if val_string is None:
+            print >>result, "Invalid or nil Data object"
+            return
     else:
```

Comment:
Add another else block for when val.GetTypeName() == "Swift.Optional<Foundation.Data>"